lib/phantomjs/platform.rb

```diff
@@ -9,10 +9,6 @@ def architecture
         RbConfig::CONFIG['host_cpu']
       end
 
-      def temp_path
-        ENV['TMPDIR'] || ENV['TEMP'] || '/tmp'
```

Comment:
I think [Dir#tmpdir](http://ruby-doc.org/stdlib-2.0.0/libdoc/tmpdir/rdoc/Dir.html#method-c-tmpdir) will be enough here.